crates/ruff_python_formatter/src/expression/expr_constant.rs

```diff
@@ -76,16 +76,10 @@ impl NeedsParentheses for ExprConstant {
     ) -> OptionalParentheses {
         if self.value.is_implicit_concatenated() {
             OptionalParentheses::Multiline
-        } else if is_multiline_string(self, context.source())
-            || self.value.is_none()
-            || self.value.is_bool()
-            || self.value.is_ellipsis()
-        {
+        } else if is_multiline_string(self, context.source()) {
             OptionalParentheses::Never
-        } else if should_use_best_fit(self, context) {
-            OptionalParentheses::BestFit
         } else {
-            OptionalParentheses::Never
+            OptionalParentheses::BestFit
```

Comment:
It's now cheap enough to always use `BestFit`. We only need to avoid it in cases where we never want parentheses